src/cli/cli.cpp

```diff
@@ -2013,6 +2014,10 @@ void Interpreter::HandleIcmpReceive(otMessage *          aMessage,
     mServer->OutputFormat("\r\n");
 
 exit:
+    if (!mPingTimer.IsRunning())
+    {
+        AppendResult(OT_ERROR_NONE);
```

Comment:
if `ping` from node 1 to node 2, node 2 is also printing `Done` because it's `HandleIcmpReceive` is called. Should make sure `aIcmpHeader->mType == OT_ICMP6_TYPE_ECHO_REPLY`